code/controllers/StaticExporter.php

```diff
@@ -232,9 +235,13 @@ public function getExportUrls() {
 		$urls = array();
 
 		foreach($classes as $obj) {
-			$link = $obj->Link;
-
-			$urls[$link] = $link;
+			if (!class_exists($obj)) {
+				continue;
+			}
+			foreach ($obj::get() as $thing) {
```

Comment:
`$thing` :-) I'd say this should be an $objInstance or something along these lines.